src/views/AboutView.vue

```diff
@@ -43,16 +43,38 @@
 				<h4>Here is a small overview from my projects</h4>
 			</div>
 
-			<div class="flip-card" v-for="project in projects" :key="project.projectTitle">
-				<div class="flip-card-inner">
-					<div class="flip-card-front">
-						<h2>{{ project.projectTitle }}</h2>
-						<h4>{{ project.projectInfo }}</h4>
+			<div class="switch-work-private">
+				<h4 @click="changeProject('private-project', 'work-project', true)" id="private-project">Private</h4>
+				<h4 @click="changeProject('private-project', 'work-project', false)" id="work-project">Work</h4>
+			</div>
+
+			<div v-if="this.switchProject" class="project-wrapper">
+				<div class="flip-card" v-for="project in projects" :key="project.projectTitle">
```

Comment:
Why not use a computed projects where u either return workProjects or projects, so u don't have to write the html twice. :)

Reply:
thx